src/components/Modal.js

```diff
@@ -0,0 +1,59 @@
+import styled from 'styled-components'
+
+const Modal = ({ state, statechange }) => {
```

Comment:
This prop name is not descriptive, please use another set of prop names

Reply:
Solved, thank you for the feedback!